### DISCUSSION TITLE
job IDs: integer or bigint?

### DISCUSSION THREAD

**Discussion by amacfie-tc** [procrastinate-org/procrastinate#869] [Q&A]
In https://github.com/procrastinate-org/procrastinate/blob/main/procrastinate/sql/schema.sql, sometimes job IDs are `integer` and sometimes they're `bigint` (or `bigserial`). Shouldn't `integer` be replaced with `bigint`?

**Comment by ewjoachim** [comment]
You're probably right ! I don't have much bandwidth to fix this now but you're welcome to make a PR :) If it's causing issues in your DB, feel free to fix it by hand in the meantime.

**Reply by ewjoachim** [reply]
Fixed in #902
